src/lcdfonts.jl

```diff
@@ -0,0 +1,256 @@
+"⟂" => "00000000000010000100001000010011111"*lcd_short,
+"≡" => "00000111110000011111000001111100000"*lcd_short,
+"±" => "00100001001111100100001000000011111"*lcd_short,
+"∓" => "11111000000010000100111110010000100"*lcd_short,
+"∇" => "00000111111000110001010100101000100"*lcd_short,
+"∂" => "00110000010000101101100011000101110"*lcd_short,
+"≠" => "00001000101111100100111110100010000"*lcd_short,
+"𝚤" => "00100000000110000100001000010100110"*lcd_short,
+" " => lcd_blank
+)
+
+const scripted_equation = "H=ħωσ_x+Jσ_y\n\nc^2=a^2+b^2\n\n|Ψ>=a^†a|ψ_1>+b^†b|ψ_2>"
+const test_string = string(
+"Keyboard characters:
+
+ABCDEFGHIJKLMNOPQRSTUVWXYZ
+abcdefghijklmnopqrstuvwxyz
+
+~!@#\$%^&*()\_+
+`1234567890-=
+
+[]\\{}|;':\",./<>?
+
+Non-keyboard characters:
+
+αβγδϵηθκλμνπρστϕχψωħ
+ΩΣΞΓΠΨΔΛΘΦ
+𝚤∞÷√±∓≠≡⟂∠°∂∇†□░█\n\n",
+length(lcd), " characters supported so far.")
+
+function lcdstring!(string_cell::Cell, str::String, pixelsize, pixelspacing; meta::Meta=GDSMeta(0,0), scripting = false, linelimit = 2^32)
+    pushstr = "push!(string_cell.refs, CellReference(s_cell, Point(pixelspacing*6*(hpos - 1), -11*pixelspacing*(vpos - offset))))"
+    hpos = 1
+    vpos = 1
+    subscript = -1
+    superscript = +1
+    for s in str
+        offset = 0.0
+        if subscript == 0
+            offset = -0.3
+            subscript = -1
+        elseif superscript == 0
+            offset = +0.3
+            superscript = +1
+        end
+        if s == '\n' || hpos > linelimit
+            vpos += 1
+            hpos = 1
+        elseif s == '_' && scripting
+            subscript = +1
+        elseif s == '^' && scripting
+            superscript = -1
+        else
+            s = string(s)
+            ss = get(lcd, s, s)
+            try
+                s_cell = drawlcdcell(ss, pixelsize, pixelspacing, meta)
+                push!(string_cell.refs, CellReference(s_cell, Point(pixelspacing*6*(hpos - 1), -11*pixelspacing*(vpos - offset))))
+            catch
+                warn("Cannot render \"", s, "\" character. Replacing with a blank character.")
+                s_cell = drawlcdcell(lcd_blank, pixelsize, pixelspacing, meta)
+                push!(string_cell.refs, CellReference(s_cell, Point(pixelspacing*6*(hpos - 1), -11*pixelspacing*(vpos - offset))))
+            end
+            hpos += 1
+        end
+        subscript -= 1
+        superscript += 1
+    end
+    string_cell
+end
+
+function drawlcdcell(code::String, pixelsize, pixelspacing, meta::Meta)
+    c = Cell(uniquename("lcd"), nm)
+    r = Rectangle(pixelsize, pixelsize) + (pixelspacing - pixelsize)/2*Point(1,1)
+    idx = 1
+    for row in 1:10
+        for col in 1:5
+            if code[idx] == '1'
+                render!(c, r + Point(pixelspacing*(col - 1), pixelspacing*(10 - row)), Rectangles.Plain(), meta)
+            end
+            idx+=1
+        end
+    end
+    c
+end
+
+function scripted_demo(;save_path = joinpath(homedir(),"Desktop"))
```

Comment:
Might consider dropping `;` so it is an optional argument rather than keyword argument